ansible/roles/ocp-workload-istio-community/tasks/wait_for_deploy.yml

```diff
@@ -6,20 +6,24 @@
 #
 # Manual Test to determine list of unready replication controllers :
 #  1) install jp :  https://github.com/jmespath/jp
-#  2) oc get rs -o json | jp 'items[?  (status.readyReplicas == ""|| status.readyReplicas == `0`) ].metadata.labels.deployment'
- 
+#  2) oc get rs -o json | jp 'items[?
+#  (status.readyReplicas == ""|| status.readyReplicas == `0`) ].metadata.labels.deployment'
+
 #  Documentation pertaining to jq syntax:
 #    - http://jmespath.org/tutorial.html
 
- 
+
 # NOTE:  requires a Deployment label of:  deployment: <deploymentname>
 
 
-- name: "Wait for following deployments to become ready: {{pod_to_wait}}"
+- name: "Wait for following deployments to become ready: {{ pod_to_wait }}"
   command: 'oc get rs -o json -n "{{ ocp_project }}"'
   register: rs_state
   changed_when: false
   retries: "{{ deploy_status_retries }}"
   delay: "{{ deploy_status_delay }}"
-  until: 'rs_state.stdout |from_json |json_query(''items[?  (status.readyReplicas == ""|| status.readyReplicas == `0`) ].metadata.labels."deployment"'') |intersect(pod_to_wait) |length == 0'
-
+  until: >-
+    rs_state.stdout | from_json | json_query(''items[?
```

Comment:
This one can be cleaned up for readability with line breaks in more natural locations.

Reply:
Also, the `''` quote is broken on this one. That is an escaped `'` from previous but does not belong in a `>-` block.